### DISCUSSION TITLE
Welcome to FLEX Discussions!

### DISCUSSION THREAD

**Discussion by NSExceptional** [Flipboard/FLEX#490] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Treat others with respect.

Feel free to discuss anything below or ask a question in a new discussion.

**Comment by BrianValente** [comment]
Hello! I just wanted to ask if an update to libFLEX will be released. Currently I'm using the version 4.4.0 from https://nscake.github.io but the Keychain option crashes the app. Thank you!
